3기/LeeGaYeon/week1/page1/styles/style.css

```
@@ -0,0 +1,81 @@
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
+
+#wrap {
+  display: flex;
+  flex-direction: column;
+  max-width: 960px;
+  width: 100%;
+  gap: 16px;
+}
+
+#container {
+  display: grid;
+  /* grid-template-columns: repeat(4, 1fr); */
+  /* sidebar 200px 고정, main은 가변 (어느 영역이 줄어드는지 파악하는게 중요)*/
+  grid-template-columns: 200px 1fr;
+  min-height: 668px;
+  width: 100%;
+  gap: 16px;
+}
+
+h1,
+h2,
+p {
+  font-family: 'Spoqa Han Sans Neo';
+  font-size: 20px;
+  line-height: 25px;
+  padding: 32px;
+}
+
+.box-item {
+  background-color: #d9d9d9;
+}
+
+article {
+  background-color: #ffffff;
+}

```

hex 코드는 줄일 수 있다면 줄여주는게 조금 더 보기 편해지기도 하고
입력하는 횟수가 줄어서 손가락이 편해져요 😁

```suggestion
article {
  background-color: #fff;
}
```